tests/rules/test_quoted_strings.py

```diff
@@ -17,11 +17,47 @@
 from tests.common import RuleTestCase
 
 
+YAML = """---
+boolean1: true
+number1: 123
+string1: foo
+string2: "foo"
+string3: "true"
+string4: "123"
+string5: \'bar\'
+string6: !!str genericstring
+string7: !!str 456
+string8: !!str "quotedgenericstring"
+binary: !!binary binstring
+integer: !!int intstring
+boolean2: !!bool boolstring
+boolean3: !!bool "quotedboolstring"
+"""
+
+
+YAML_MULTILINE = """---
+multiline string 1: |
+  line 1
+  line 2
+multiline string 2: >
+  word 1
+  word 2
+multiline string 3:
+  word 1
+  word 2
+multiline string 4:
+  "word 1\\
+   word 2"
+---
+"""
+
+
```

Comment:
@adrienverge Extracted the two yaml strings so they can be reused and each test becomes cleaner.

Reply:
I understand you removed them to avoid code duplication, but having source YAML directly above expected errors really helps reading tests, and checking everything works as expected.

Unless you really really prefer like that, can you please revert this change?

Reply:
I honestly was divided between the two approaches and ended up choosing the one with less code duplication because the testes seemed cleaner to me. But I understand it's something completely opinionated and... you should be the one playing the preference card. ;)